Entities/Items/Projectiles/Arrow.as

```diff
@@ -484,6 +485,7 @@ f32 ArrowHitBlob(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlo
 
 		// check if shielded
 		const bool hitShield = (hitBlob.hasTag("shielded") && blockAttack(hitBlob, velocity, 0.0f));
+		const bool hitKeg = hitBlob.getName() == "keg";
```

Comment:
Put the condition in brackets like with `hitShield` for consistency, KAG does this (almost) always when setting a variable to a condition.

Reply:
resolved